scripts/plugins/support-pssl.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+#
+# Copyright (c) Citrix Systems 2008. All rights reserved.
```

Comment:
So for the new script , about the copyright ,  I think I should remove that , right?